src/data/scripts/hullmods/vayra_sometimes_war.java

```diff
@@ -45,15 +45,14 @@ public class vayra_sometimes_war extends BaseHullMod {
         FUEL_CAP.put(HullSize.CRUISER, 60f);
         FUEL_CAP.put(HullSize.CAPITAL_SHIP, 150f);
     }
-
     private static final Map<HullSize, Float> FUEL_EFF = new HashMap<>();
 
     static {
         FUEL_EFF.put(HullSize.FIGHTER, 0f);
         FUEL_EFF.put(HullSize.FRIGATE, 0.5f);
         FUEL_EFF.put(HullSize.DESTROYER, 1f);
-        FUEL_EFF.put(HullSize.CRUISER, 2f);
-        FUEL_EFF.put(HullSize.CAPITAL_SHIP, 5f);
+        FUEL_EFF.put(HullSize.CRUISER, 1.5f);
+        FUEL_EFF.put(HullSize.CAPITAL_SHIP, 2.5f);
```

Comment:
again, why cripple these constants? what does "git blame" say about these lines? does the commit where they're introduced say anything good?